client/src/modules/user/model/reducer.js

```diff
@@ -21,6 +21,15 @@ const currentUser = (state = null, action) => {
 
       return state;
     }
+    case AuthorizationActionTypes.LEAVE_PROFILE_VIEW_SUCCESS: {
```

Comment:
We have already action for this - a common action  `clearMetaDataSuccess` maybe we can use it. It Is dispatched on every route change and some reducers react when this action is dispatched. If you use this action only reducer file would be changed. `currentUser` reducer may react on it and remove unnecessary data.

Reply:
you're right